File: javascript/Day_1/Question_12.js
number = 5;

const check = (number) => {
  for (let i = 2; i < number; i++) {
    if (number % i == 0) {
      if (number === 1) {
        break;
      }
      console.log(`${number} is not prime number`);
      return;
    }
  }
  console.log(`${number} is prime number`);
};

const result = check(number);

File: dsa/Day_1/Question_12.cpp
#include <iostream>
using namespace std;

void check(int number){
    if(number <= 1){
        cout << number << " is not prime number" << endl; 
    }
    for (int i = 2; i < number; i++)
    {
        if(number % i == 0){
            cout << number << " is not prime number" << endl;
            return;
            break;
        }
        
    }
    cout << number << " is prime number" << endl;
}

int main(){
    int number;

    cout << "number: ";
    cin >> number;

    check(number);
}
